hawkular-alerts-engine/src/main/java/org/hawkular/alerts/engine/impl/DbDefinitionsServiceImpl.java

```diff
@@ -196,7 +197,10 @@ private void initFiles(String folder) {
                 initNotifiers(initFolder);
             }
         } catch (Exception e) {
-            msgLog.errorDatabaseException("Error initializing files. Msg: " + e.getMessage());
+            if (log.isDebugEnabled()) {
```

Comment:
A small (a non-important comment). debugf method invokes the isDebugEnabled() so you can save the check outside in the code.